lib/commands/commandCommitMessageHook.js

```diff
@@ -12,7 +12,7 @@ exports.builder = function (yargs) {
     yargs.option('config', {
         alias: 'c',
         desc: 'Config file path',
-        default: Config.defaultPath,
+        default: fs.existsSync(Config.defaultPath) ? Config.defaultPath : Config.templatePath,
```

Comment:
We should move this exist check into the global hook template e.g.
```shell
#!/bin/sh
if [ -f ./git-conventional-commits.json ]
then
  git-conventional-commits commit-msg-hook "$1"
fi

Reply:
This is a great idea for global usage - but will override the actual setting-per-repo logic.
I will check some other options and reply.

Reply:
Why do you think it will override the per repo settings?

Reply:
you're right. Thought the hook runs ins scope of the hook folder. But then we don't have a global file to use :D

Reply:
I would run the command every time - if the file exists or not. 
If no file exists and no global file f.e. `~/.git-conventional-commits/default.json` exists, the hook doesn't anything.
If the file exists in the repository we use this. 
If no local file exists, use the global file above.

Think this would be kind of standard behavior and match many possibilities.
If you agree, i would change the logic.

Reply:
Sounds good to me.

Reply:
However let's implement this logic within the hook rather than in the app itself.